### PR TITLE
Increase filter by spread speed

### PR DESCRIPTION
**Pull request by FelixKleinpeter**
Because I needed to use it a lot, I applied small changes on the `skip_entry_by_spread` function so now its faster to use.

### CODE REVIEW

**Comment by FelixKleinpeter** [comment]
Here are the exploration and results for the best values of `max_spread`. Please scroll at the end.

https://alpacajp.slack.com/files/UK59L1VTJ/FLZG2V8GZ/evaluation_spread_filtered.html

**Comment by FelixKleinpeter** [comment]
Yuki helps me for this PR. We observed there was no correlations between spread and loss. Here is a scatter plot of the loss for each centile of spread :
spread_pl_centiles

The only things which makes me think there was a correlation is that for the 100th centil there is mostly a profit of 0. Here is the plot of profit by spread : 
spread_pl

What do you think? Should we apply the function on this extreme points or should we leave it like that? Thanks!

**Comment by FelixKleinpeter** [comment]
spread_value

Value of portfolios with only EURUSD of 95% confidence by spread threshold. 
-1 is the value of the not filtered portfolio.

**Comment by exilis** [comment]
@FelixKleinpeter spread can take different range of values based on hour of the day. So I would recommend you to break it down into hour by hour analysis.

**Comment by FelixKleinpeter** [comment]
Thanks for your advice. Here are the results for an hour plotting (end of the file) : 
https://alpacajp.slack.com/archives/CK8ST5ZFZ/p1565771297079300

**Comment by FelixKleinpeter** [comment]
What should we do with this one? Considering the results, is it necessary to merge it?

File: src/backlight/strategies/filter.py [hunk 1]
```diff
@@ -49,11 +49,14 @@ def skip_entry_by_spread(
...
     spread = mkt.spread
-
     deleted_ids = []  # type: List[int]
-    for i in trades.ids:
-        entry = trades.get_trade(i).index[0]
+
+    sorted_by_ids = (
+        trades.reset_index().sort_values(by=["_id", "index"]).set_index("index")
+    )
+    for i in range(0, sorted_by_ids.index.size, 2):
```
Comment (keisuke-umezawa) [inline src/backlight/strategies/filter.py]: Here, you assumed that each trade always contains entry and exit? It is not always true.

**Pull request closed without merging**